src/main/io/ledstrip.h

```diff
@@ -168,6 +175,11 @@ typedef struct ledCounts_s {
     uint8_t ringSeqLen;
 } ledCounts_t;
 
+typedef struct ledBarState_s {
```

Comment:
`uint8_t` array for total led count would do. And in can be `static` in ledstrip.c